resolwe_bio/processes/import_data/pathway_map.yml

```diff
@@ -0,0 +1,34 @@
+- slug: upload-metabolic-pathway
+  name: Metabolic Pathway File
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/legacy:latest
+    resources:
+      network: true
+  data_name: Metabolic Pathway
+  version: 0.1.0
+  type: data:pathway
+  category: upload
+  persistence: RAW
+  scheduling_class: interactive
+  description: >
+    Upload pathway json.
+  input:
+    - name: upload
```

Comment:
We usually use name `src` for files that need to be uploaded. This is not mandatory, but it is nice if all the processes share the same naming convention.